### PR TITLE
Updated footer and added lang attribute to html tag

### PR DESCRIPTION
**Pull request by KhalidAdan**
Adds some padding and finalized the footer to core GC design specs. Also adds a `lang` attribute to the html tag!

This encompasses the work outlined in [SAEB-1108](https://jira-dev.bdm-dev.dts-stn.com/browse/SAEB-1108) and adds localized paths for languages (not full translation implementation)!

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
 module.exports = {
+  i18n: {
+    locales: ['en', 'fr'],
+    defaultLocale: 'en',
+    localDetection: true,
```
Comment (JeremyKennedy) [inline next.config.js]: should this be `localDetection` or `localeDetection`?

File: components/Layout/Footer.tsx [hunk 2]
```diff
@@ -63,39 +63,64 @@ export const Footer = () => (
...
+          width="200px"
+          height="40px"
+          src="/wmms-blk.svg"
+          alt="Symbol of the Government of Canada"
+        />
+      </div>
+    </div>
+    <div className="flex justify-between">
+      <div className="flex items-center md:hidden py-8 md:pt-0">
+        <p className="px-4">Top of page</p>
```
Comment (JeremyKennedy) [inline components/Layout/Footer.tsx]: This says "top of page" but I don't see that anywhere in the page, it this text supposed to be visible somewhere?

**Review by JeremyKennedy — CHANGES_REQUESTED** [review]
- [x] The spacing in the footer is too high (screenshots sent in Teams)

File: components/Layout/Footer.tsx (hunk 2 above)
Comment (KhalidAdan) [reply components/Layout/Footer.tsx]: Visible on mobile only, as in the GC design mockups. That's what the `md:hidden` is doing

File: components/Layout/Footer.tsx [hunk 3]
```diff
@@ -1,101 +1,128 @@
...
-            src="/wmms-blk.svg"
-            alt="Symbol of the Government of Canada"
-          />
-        </div>
+    <div className="flex justify-between px-4 py-8 sm:container mx-auto">
+      <ul className="flex flex-col md:flex-row md:space-x-6 list-disc px-4 text-[#284162]">
+        <li>
+          <a href="#">Social media</a>
+        </li>
+        #284162
```
Comment (JeremyKennedy) [inline components/Layout/Footer.tsx]: image
This doesn't quite seem right! 😉

**Review by JeremyKennedy — CHANGES_REQUESTED** [review]
Looks better, except for the one issue with "social media" at the bottom!

File: components/Layout/Footer.tsx (hunk 3 above)
Comment (JeremyKennedy) [reply components/Layout/Footer.tsx]: It's better, but you should still remove the bullet point before Social Media, see this screenshot from canada.ca:
image

**Review by JeremyKennedy — CHANGES_REQUESTED** [review]
Still need to remove bullet point before Social Media

File: components/Layout/Footer.tsx (hunk 3 above)
Comment (JeremyKennedy) [reply components/Layout/Footer.tsx]: Also, not that it's a problem, but canada.ca has an off-white color, unlike ours which is just plain white.

**Comment by KhalidAdan** [comment]
> Still need to remove bullet point before Social Media

Yikes, I missed all these, my fault! I completelyyy misunderstood and blame sleep deprivation!

**Review by JeremyKennedy — APPROVED** [review]
> Yikes, I missed all these, my fault! I completelyyy misunderstood and blame sleep deprivation!

Whoops! I thought I mentioned the bullet point, turns out I didn't so definitely my bad there! 😃 

Looks good! And I checked the mobile layout to find the Top of page button. 👍

**Pull request merged**